docs/getting-started/release-notes/v2_13_0.md

```diff
@@ -0,0 +1,110 @@
+# Version 2.13.0 (December 2023)
+
+Welcome to the Zowe Version 2.13.0 release!
+
+See [New features and enhancements](#new-features-and-enhancements) for a full list of changes to the functionality. See [Bug fixes](#bug-fixes) for a list of issues addressed in this release.
+
+**Download v2.13.0 build**: Want to try new features as soon as possible? You can download the v2.13.0 build from [Zowe.org](https://www.zowe.org/download.html).
+
+## New features and enhancements
+
+Zowe Version 2.13.0 contains the enhancements that are described in the following topics.
+
+### Zowe installation and packaging
+
+### Zowe Application Framework
+
+### Zowe API Mediation Layer
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Incorporated all source code from the `zowe/imperative` Github repository into the `zowe/zowe-cli` repository for a more streamline code architecture. This change should have no user impact.
+- Deprecated `getDataSet` in the `zosfiles` command group utility functions. Use `zosfiles` SDK's `ZosFilesUtils.getDataSetFromName` command instead. ([#1696](https://github.com/zowe/zowe-cli/issues/1696))
+
+#### z/OS FTP Plug-in for Zowe CLI
+
+- Added a step to check the validity of USS file path for the `upload` and `stdin-to-uss-file` commands. ([#145](https://github.com/zowe/zowe-cli-ftp-plugin/pull/145))
+
+### Zowe Explorer
+
+#### Zowe Explorer (Core)
+
+- See the [Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer API
+
+- See the [Zowe Explorer API](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-api/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer FTP Extension
+
+- See the [Zowe Explorer FTP Extension](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/zowe-explorer-ftp-extension/CHANGELOG.md) changelog for updates included in this release.
+
+#### Zowe Explorer ESLint Plug-in
+
+- See the [Zowe Explorer ESLint Plug-in](https://github.com/zowe/vscode-extension-for-zowe/blob/main/packages/eslint-plugin-zowe-explorer/CHANGELOG.md) changelog for updates included in this release.
+
+## Bug fixes
+
+Zowe Version 2.13.0 contains the bug fixes that are described in the following topics.
+
+### Zowe installation and packaging
+
+### Zowe Application Framework
+
+### Zowe API Mediation Layer
+
+### Zowe CLI
+
+#### Zowe CLI (Core)
+
+- Added missing npm-shrinkwrap.
```

Comment:
I would use this link: https://github.com/zowe/zowe-cli/pull/1978

I am not totally sure what happened with the shrinkwrap. It might be clearer to say something like: "Added missing npm-shrinkwrap.json file to package.json".